gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -711,7 +711,7 @@ def __init__(self,
     self._header = headers and '\n'.join([h.strip() for h in headers]) + '\n'
 
   def expand(self, pcoll):
-    return pcoll | textio.WriteToText(
+    return pcoll | 'Write to VCF' >> textio.WriteToText(
         self._file_path,
         append_trailing_newlines=False,
         num_shards=self._num_shards,
```

Comment:
FYI: `num_shards` is the number of output files. The default is 1, and we expect the output will be only one VCF file.